tests/test_browser.py

```diff
@@ -187,10 +186,9 @@ def test_zzz_html_source_map(self):
 
   @no_wasm_backend('wasm source maps')
   def test_emscripten_log(self):
-    # TODO: wasm support for source maps
+    # TODO: wasm support for source maps. emscripten_loadSourceMap looks at $HTML.map but it should be $NAME.wasm.map.
```

Comment:
Is `emscripten_load` supposed to be here?

Reply:
Yes, `emscripten_loadSourceMap` is an internal API that has the problem described here.